x-pack/plugins/lens/public/datatable_visualization/_visualization.scss

```
@@ -1,3 +1,15 @@
 .lnsDataTable {
   align-self: flex-start;
 }
+
+.lnsDataTable__filter {
+  opacity: 0;
+  transition: opacity $euiAnimSpeedNormal ease-in-out;
+}
+
+.lnsDataTable__cell:hover,
+.lnsDataTable__cell:focus-within {
+  .lnsDataTable__filter {
+    opacity: 1;
+  }
+}
```

Sorry I actually messed up with this one. We don't want anything focused in the cell to show the filters, we want a focused **filter** button to increase the opacity. So this block should actually be:

```suggestion
.lnsDataTable__cell:hover .lnsDataTable__filter,
.lnsDataTable__filter:focus-within {
  opacity: 1;
} 
```